packages/image_picker/image_picker_android/README.md

```diff
@@ -7,5 +9,17 @@ The Android implementation of [`image_picker`][1].
 This package is [endorsed][2], which means you can simply use `image_picker`
 normally. This package will be automatically included in your app when you do.
 
+## Photo Picker
+
+This package has optional Android Photo Picker functionality.
+
+To use this feature, add these lines to your Flutter app:
```

Comment:
How about "add the following code to your app before calling any `image_picker` APIs:"